tests/components/homekit/test_type_switches.py

```diff
@@ -240,19 +239,12 @@ async def test_vacuum_set_state(hass, hk_driver, events):
     assert events[-1].data[ATTR_VALUE] is None
 
 
-@pytest.mark.parametrize(
-    "entity_id, attrs",
-    [
-        ("scene.test", {}),
-        ("script.test", {}),
-        ("script.test", {ATTR_CAN_CANCEL: False}),
```

Comment:
Does it make sense to keep the tests with this behavior?  (Assume all scripts can cancel?)

Reply:
taken care of